.github/actions/unit-test-setup/action.yml

```diff
@@ -1,25 +1,11 @@
 name: Setup unit test
 description: Download Maven caches needed for unit tests
 
-inputs:
```

Comment:
Should leave the inputs, and pass on to `actions/java-setup` as it should be possible to use `unit-test-setup` with a different JDK version if needed